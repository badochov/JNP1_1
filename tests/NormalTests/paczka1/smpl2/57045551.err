Error in line 1: 9ikACsp8g S758 25,5
Error in line 4: 9ikACsp8g A919 27,7
Error in line 22: JndXXq A120 89,4
Error in line 31: BcJqtnWTX5s A120 40,9
