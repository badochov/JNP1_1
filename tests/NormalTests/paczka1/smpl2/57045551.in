9ikACsp8g S758 25,5
?
A120 S758 55,9
9ikACsp8g A919 27,7
xWUSWHhJv A120 10,1
?3eHl9iBxJR
xWUSWHhJv A120 29,7
A120 S758 20,1
?9ikACsp8g
9ikACsp8g A120 25,3
9ikACsp8g A120 41,2
3eHl9iBxJR S697 51,5
3eHl9iBxJR S697 5,7
xWUSWHhJv A120 1,4
9ikACsp8g A120 28,4
?TXd0h
A120 S758 51,5
TXd0h S697 21,6
A120 S758 219,4
xWUSWHhJv A120 2,0
TXd0h S697 46,2
JndXXq A120 89,4
xWUSWHhJv A919 49,0
?
JndXXq S697 46,4
TXd0h S697 94,6
?3eHl9iBxJR
9ikACsp8g A120 82,9
JndXXq S697 83,5
?BcJqtnWTX5s
BcJqtnWTX5s A120 40,9
9ikACsp8g S697 80,4
9ikACsp8g S697 188,1
3eHl9iBxJR A919 8,2
TXd0h S697 187,3
9ikACsp8g A120 78,5
BcJqtnWTX5s S697 61,4
xWUSWHhJv A919 178,8
BcJqtnWTX5s S697 160,3
?