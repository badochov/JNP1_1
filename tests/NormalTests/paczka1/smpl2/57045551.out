3eHl9iBxJR S 45,8
9ikACsp8g A 15,9
A120 S 203,7
TXd0h S 24,6
xWUSWHhJv A 20,2
A120 36,1
S697 70,4
S758 203,7
3eHl9iBxJR S 45,8
3eHl9iBxJR S 45,8
9ikACsp8g A 70,4 S 107,7
A120 S 203,7
BcJqtnWTX5s S 98,9
JndXXq S 37,1
TXd0h S 117,3
xWUSWHhJv A 150,0
A120 90,6
S697 406,8
S758 203,7
A919 129,8
